cmd/interchaincmd/tokentransferrercmd/deploy.go

```diff
@@ -578,12 +577,11 @@ func CallDeploy(_ []string, flags DeployFlags) error {
 		remoteSupply  *big.Int
 	)
 
-	tokenRemoteDecimals := tokenHomeDecimals
-	if flags.remoteFlags.tokenDecimals != 0 {
-		tokenRemoteDecimals = flags.remoteFlags.tokenDecimals
-	}
-
 	if !flags.remoteFlags.native {
+		tokenRemoteDecimals := tokenHomeDecimals
```

Comment:
So for ERC20TokenRemote, we default the `tokenDecimals` to be the same as the `tokenHomeDecimals`, but allow this to be overridden by a user provided flag? This seems perfectly reasonable to me, could we just add a comment here documenting the behavior?